### DISCUSSION TITLE
will frigate+ provide models that can be used on tensorrt?

### DISCUSSION THREAD

**Discussion by vajonam** [blakeblackshear/frigate#7917] [Ask A Question] [answered]
At the moment it appears that friage+ models will support coral to start, are there plans to support tensorrt with frigate+?

**Comment by NickM-27** [comment]
To be clear I don't work on Frigate+ so this is not an official answer, but from what I understand on https://deploy-preview-6262--frigate-docs.netlify.app/plus/#supported-model-types it seems openvino will be next.

This is purely speculation on my part, but based on the relatively low download numbers for the tensorrt beta images https://github.com/blakeblackshear/frigate/pkgs/container/frigate/versions?filters%5Bversion_type%5D=tagged&page=1 I would guess the ROI would be less than supporting other more popular detectors.

**Reply by blakeblackshear** [reply]
This is correct, I do plan to try and support tensorrt. It's lower on the priority list due to the smaller number of users.

**Comment by NickM-27** [comment, marked as answer]
This is now supported https://docs.frigate.video/plus/#supported-detector-types
